src/gui/newwizard/logging.cpp

```diff
@@ -0,0 +1,3 @@
+#include "logging.h"
```

Comment:
I guess we should have more than one category and thus we won't need the logging.h/cpp

Reply:
As the category is only used in the .cpp then we won't need to make them available in the header with Q_DECLARE_LOGGING_CATEGORY

Reply:
We could also look on https://api.kde.org/ecm/module/ECMQtDeclareLoggingCategory.html

Reply:
We can use more specific categories.